src/splat/disassembler/wasm_disassembler.py

```diff
@@ -0,0 +1,14 @@
+from . import disassembler
+from ..util import log, compiler
+from ..util.options import SplatOpts
+from typing import Set
+
+class WasmDisassembler(disassembler.Disassembler):
+    def configure(self, opts: SplatOpts):
+        print("Configuring WASM Disassembler!")
```

Comment:
If there's nothing to do in these functions, I think it'd be better to just `pass` in them than have these prints

Reply:
After thinking for a sec, I realize that the intent is probably to functionality here lol